DependencyInjection/SonataAdminExtension.php

```diff
@@ -85,8 +85,23 @@ public function load(array $configs, ContainerBuilder $container)
         $processor = new Processor();
         $config = $processor->processConfiguration($configuration, $configs);
 
-        $config['options']['javascripts'] = $config['assets']['javascripts'];
-        $config['options']['stylesheets'] = $config['assets']['stylesheets'];
+        $javascripts = array_unique(
+            array_merge(
+                $config['assets']['javascripts'],
+                $config['assets']['extra_javascripts']
```

Comment:
What if the `extra_javascripts` configuration is not defined ? Does this somehow default to `[]` ?